wagtail/contrib/forms/models.py

```diff
@@ -121,6 +122,15 @@ class AbstractFormField(Orderable):
         FieldPanel("default_value", classname="formbuilder-default"),
     ]
 
+    api_fields = [
+        APIField("label"),
+        APIField("help_text"),
+        APIField("required"),
+        APIField("field_type"),
+        APIField("choices"),
+        APIField("default_value"),
+    ]
```

Comment:
@saevarom  thanks for updating this but would you be able to add a unit test that covers this also?

Reply:
@saevarom Greetings, Could I volunteer to write the unit tests, as I am new to the repository it would be a great learning experience.

Reply:
@saevarom Hi! I inserted the above code but I can't expose the form fields in the API. This is the response I am getting.

```{

            "id": 6,
            "meta": {
                "type": "home.FormPage",
                "detail_url": "http://localhost/api/v2/pages/6/",
                "html_url": "http://localhost/i-am-suyash/",
                "slug": "i-am-suyash",
                "first_published_at": "2022-06-28T16:15:39.791338Z"
            },
            "title": "I am Suyash"
}
```
I have added api_fields to FormPage.

Reply:
@saevarom Hi! I have made quite a bit of progress since we talked. I have successfully set up FormPage and check how APIFields work. What should be my startegy to write the tests?

Do I need to add fixtures to add the model data and write a test against that...  

or 

Should I manually create a FormPage in the Django code itself and check if APIFields is working properly or not?

Reply:
@GeekGawd have you checked out the wagtail codebase and have you successfully run the test suite?

This is a bit tricky since I think the Wagtail v2 API does not have any form pages to start with. I probably would add a FormPage to the wagtail/test/demosite/fixtures/demosite.json and then write tests against that type of page being queried in the API.

If you feel like this is not the best first issue for you to learn from then I can take over, just let me know.

Reply:
@GeekGawd what's the status on your PR? I was wondering if I should perhaps incorporate your changes into this PR?
I would label you as the author of the commit so you would get the mention for the code :)